docs/rst/notes/notes.rst

```diff
@@ -4,32 +4,27 @@
 Notes
 #####
 
-.. include:: forthcoming_version.rst
+..
+    TODO uncomment when there are forthcoming notes
+    .. include:: forthcoming_version.rst
 
 ##############
-Version v1.3.0
+Version v1.4.0
 ##############
 
-This release includes the following **bug fixes**:
+This release includes the following **features**:
 
-* Fix error in y-axis resize that went to infinite.
-
-This release includes the following new **internal changes**:
-
-* Upgrade
-  `Fast DDS v2.8.0 <https://github.com/eProsima/Fast-DDS/releases/tag/v2.8.0>`_
-  to
-  `Fast DDS v2.9.0 <https://github.com/eProsima/Fast-DDS/releases/tag/v2.9.0>`_.
-
-* Upgrade
-  `Fast DDS Statistics Backend v0.7.1 <https://github.com/eProsima/Fast-DDS-statistics-backend/releases/tag/v0.7.1>`_.
-  to
-  `Fast DDS Statistics Backend v0.8.0 <https://github.com/eProsima/Fast-DDS-statistics-backend/releases/tag/v0.8.0>`_.
+* Dump the information from the database to a JSON file.
+* Remove inactive entities from the database.
+* Delete the statistics data of all the entities.
+* Schedule remove old data every ``x`` sec.
```

Comment:
write `seconds`. I would also change sentence (in user documentation let's try to be as formal as possible)